src/iop/exposure.c

```diff
@@ -504,6 +555,7 @@ void gui_update(struct dt_iop_module_t *self)
 
   dt_bauhaus_combobox_set(g->mode, g_list_index(g->modes, GUINT_TO_POINTER(p->mode)));
 
+  gtk_toggle_button_set_active(GTK_TOGGLE_BUTTON(g->compensate_exposure_bias), p->compensate_exposure_bias);
```

Comment:
This must be done conditionally based on the setting, no?

Reply:
the setting is the second argument of the function (p->compensate_exposure_bias). Or I don't understand your question ?

Reply:
Right, the setting is the second argument... I don't understand **MY** question :) Sorry. There is a bug in there because it is never reset (see report above) but the issue is probably not there :)